src/input.service.ts

```diff
@@ -264,8 +272,16 @@ function isForForm(direction: Direction, selected: Element): boolean {
 @Injectable()
 export class InputService {
 
-  private inputPane = Windows.UI.ViewManagement.InputPane.getForCurrentView();
-  public keyboardVisible = new BehaviorSubject(false);
+  /**
+   * Inputpane and boolean to indicate whether it's visible
+   */
+  private inputPane = (<any>window).Windows ? Windows.UI.ViewManagement.InputPane.getForCurrentView() : null;
+  public keyboardVisible = (<any>window).Windows ? new BehaviorSubject(false) : null;
+
+  /**
+   * Default trigger navigation element
+   */
+  public defaultTrigger: TriggerNavigable = null;
```

Comment:
Since this is not used anywhere in arcade-machine itself, I think having the frontend manage it would be better; arcade-machine doesn't care about it at all.